xml/dhcp_vendor_class.xml

```diff
@@ -203,4 +212,80 @@
     <param pos="0" name="os.family" value="Windows"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^(?i)Cisco Systems Inc. Wireless Phone (\d{4})$">
+    <description>Cisco Wireless Phone</description>
+    <example hw.model="7925">Cisco Systems Inc. Wireless Phone 7925</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.certainty" value="0.8"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(?i)Cisco Systems, Inc. IP Phone (.*)">
```

Comment:
1. Escape the period in `Inc\.` otherwise we'll match any character in that position.
2. Tighten the regex rather than allowing unbounded matches that may result in unexpected matches and captures as well as possible performance issues. Is there a reason we couldn't use something like `(CP-\d+G?)` instead of `(.*)`? Note, I derived the pattern from this [existing sip banners fingerprint](https://github.com/rapid7/recog/blob/8383440f954a4511463fd256d62b546ce51075cd/xml/sip_banners.xml#L48).
3. Is there any reason not to have an end of string anchor?

Reply:
I added a size limit and an end-of-string anchor. I don't see any benefit in making the model regexp more complicated (something like `CP-?\d{4}(?:G|G-GE)?`), since that would decrease readability and performance at the same time.